pallets/connectors/src/lib.rs

```diff
@@ -486,3 +547,28 @@ pub mod pallet {
 		}
 	}
 }
+
+#[cfg(test)]
+mod tests {
+	use cfg_primitives::AccountId;
+	use sp_runtime::traits::AccountIdConversion;
+
+	use super::DomainAddress;
+
+	#[test]
+	fn domain_address_account_derivation() {
+		assert_eq!(
+			account_from(DomainAddress::EVM(1284, [9; 20])),
```

Comment:
I just added these tests to verify that the account derivation was taking the inner address into account. We probably can drop these but leave it up to the reviewers.